### PR TITLE
fix: Inconsistent translation format for tooltips

### PR DESCRIPTION
**Pull request by Xhy-5000**
Fix: https://github.com/casdoor/casdoor/issues/1309

### CODE REVIEW

**Comment by hsluoyz** [comment]
@imchell plz review

**Comment by imchell** [comment]
You can update **all** i18n data with [one command](https://casdoor.org/docs/contributing#how-to-update-i18n-data) so that you do not have to add translation entries to different `json` files in `web/src/locales/` manually.

**Comment by Xhy-5000** [comment]
> 
Oh I see that, thx.

File: web/src/locales/en/data.json [hunk 1]
```diff
@@ -478,119 +478,119 @@
...
     "Bucket": "Bucket",
-    "Bucket - Tooltip": "Bucket - Tooltip",
+    "Bucket - Tooltip": "Bucket name",
     "Can not parse Metadata": "Can not parse Metadata",
     "Can signin": "Can signin",
     "Can signup": "Can signup",
     "Can unlink": "Can unlink",
     "Category": "Category",
-    "Category - Tooltip": "Category - Tooltip",
+    "Category - Tooltip": "Category",
```
Comment (hsluoyz) [inline web/src/locales/en/data.json:499]: Don't just remove `- Tooltip` as final text

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**